src/io/rkn/conformity.clj

```diff
@@ -56,32 +56,53 @@
   (or (some #(and (has-attribute? db %) %) [:conformity/conformed-norms default-conformity-attribute])
       :conformity/conformed-norms))
 
+(defn last-tx-instant
+  "Returns a value of the :db/txInstant attribute of the last transaction."
+  [db]
+  (let [tx (-> db (d/basis-t) (d/t->tx))]
+    (ffirst (q '[:find ?inst
```

Comment:
How do we know that the first thing returned here is the last tx-instant?

Reply:
Ah ok 

https://docs.datomic.com/on-prem/clojure/index.html#datomic.api/basis-t
> Returns the t of the most recent transaction reachable via this db value.